src/launcher/StracciatellaLauncher.cc

```diff
@@ -10,6 +10,7 @@ StracciatellaLauncher::StracciatellaLauncher() {
       { Fl_Group* o = new Fl_Group(0, 50, 520, 300, "@> Play  ");
         o->labelsize(16);
         o->labelcolor((Fl_Color)24);
+        o->hide();
```

Comment:
Just a guess, but I think you need to change to the first tab before saving and exporting to C++ in Fluid. Otherwise the tab currently open in Fluid will be open in the application. Or we just add some code that shows the first tab on startup.